BraintreeDropIn/BTCardFormViewController.m

```diff
@@ -274,6 +286,55 @@ - (void)updateRequiredFields {
     }
 }
 
+#pragma mark - card.io
+
+- (void)setupCardIO {
+    if ([self isCardIOAvailable]) {
+        BTUIKInputAccessoryToolbar *cardToolBar = [BTUIKInputAccessoryToolbar new];
+        UIBarButtonItem *flexSpace = [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemFlexibleSpace target:nil action:nil];        
+        UIBarButtonItem *cardIOButton = [[BTUIKBarButtonItem alloc] initWithTitle:BTUIKLocalizedString(SCAN_CARD_IO_ACTION) style:UIBarButtonItemStylePlain target:self action:@selector(presentCardIO)];
+        cardToolBar.items = @[flexSpace, cardIOButton, flexSpace];
+        self.cardNumberField.textField.inputAccessoryView = cardToolBar;
+    }
+}
+
+- (BOOL)isCardIOAvailable {
```

Comment:
While I think this method would work for people the majority of the time, is there a way to disable it? I don't know who would have `card.io` (maybe as another dependency) and not want it for the card form, but just something to consider. Maybe implement when someone asks?